packages/x-components/src/x-installer/x-installer/types.ts

```diff
@@ -15,7 +15,11 @@ export interface InstallXOptions<API extends XAPI = XAPI> extends XPluginOptions
    * The Vue component used as root of the application. If it is not passed, no Vue Application is
    * initialized, only plugin installed.
    */
-  app?: VueConstructor | ComponentOptions<Vue>;
+  rootComponent?: Component;
+  /**
+   * The Vue application instance. If it is not passed, a new Vue application is created.
+   */
+  app?: App;
```

Comment:
It would be a breaking change, wouldn't it? 
I'm thinking if it makes sense to continue giving support to Vue app creation through options. It not, maybe we could keep `app` option as root component over mounting and creation the Vue application

Reply:
As discussed in a call, we will remove the app from the `InstallXOptions`, as we don't need it anymore because the app creation with Vue3 is more flexible, and several instances of an app could be created.